Controller/RegistrationController.php

```diff
@@ -96,7 +96,7 @@ public function registerAction(Request $request)
     /**
      * Tell the user to check his email provider.
      */
-    public function checkEmailAction()
+    public function checkEmailAction(Request $request)
```

Comment:
I explained it all [here](https://github.com/sonata-project/SonataNewsBundle/pull/342#issuecomment-255391982)
**TL;DR** : you don't need to add a $request parameter on every action for it to be "extensible"